backend/dataall/modules/datasets_base/__init__.py

```diff
@@ -0,0 +1,18 @@
+from typing import Set
+from dataall.base.loader import ModuleInterface, ImportMode
+
+
+class DatasetBaseModuleInterface(ModuleInterface):
+    @staticmethod
+    def is_supported(modes: Set[ImportMode]) -> bool:
+        supported_modes = {
```

Comment:
could supported_modes just be `ImportMode.all()` ?

Reply:
I see it is not used for `CDK_CLI_EXTENSION`

Reply:
I did not want to just give a wildcard of import; but as I explained in my comment, the result is not very pretty